docs/HowTo/clusterconnectivity/aiven-karapace-cluster-ssl-protocol.md

```diff
@@ -45,7 +45,7 @@ registry cluster with Klaw using SSL protocol:
    Environment** and enter the details to add your schema registry
    environment. Click **Save**.
 7. Open the `application.properties` file for [cluster-api]
-   (klaw/cluster-api/src/main/resources) modules.
+   (Klaw/cluster-api/src/main/resources) modules.
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?